flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -1434,12 +1428,6 @@
 	@Deprecated
 	public static final int DEFAULT_TASK_MANAGER_MEMORY_SEGMENT_SIZE = 32768;
 
-	/**
-	 * The implementation to use for spillable/spilled intermediate results, which have both
-	 * synchronous and asynchronous implementations: "sync" or "async".
-	 */
-	public static final String DEFAULT_TASK_MANAGER_NETWORK_DEFAULT_IO_MODE = "sync";
```

Comment:
This is part of the public API (since `ConfigConstants` is `@Public`) and cannot be removed.